app/src/main/java/com/vincent_falzon/discreetlauncher/core/ApplicationsList.java

```diff
@@ -109,6 +110,7 @@ public void update(Context context)
 		// Retrieve the list of user profiles
 		UserManager userManager = (UserManager)context.getSystemService(Context.USER_SERVICE) ;
 		List<UserHandle> userProfiles = userManager.getUserProfiles() ;
+		PackageManager packageManager = context.getPackageManager() ;

```

Comment:
```suggestion
		PackageManager apkManager = context.getPackageManager() ;
```

Just noticed that for some reasons I don't remember, I called the package manager "apkManager" in several other places of the launcher, let's do the same here for consistency.